### ISSUE TITLE
Suggestion: create V0 tags for tflinters

### ISSUE THREAD

**Issue by audunsolemdal** [terraform-linters/tflint#2462]
## Introduction

My org sets up tflint and separate .tflint.hcl across multiple repositories

Essentially I would like to have the option to set

```hcl
plugin "azurerm" {
  enabled = true
  version = "v0"
  source  = "github.com/terraform-linters/tflint-ruleset-azurerm"
}
```
Since the tag is not available, we end up with this

Failed to install a plugin; Failed to fetch GitHub releases: GET https://api.github.com/repos/terraform-linters/tflint-ruleset-azurerm/releases/tags/v0 404 Not Found []

## Proposal

My suggestion would be to provide a v0 tag along with the regular tags (e.g. 0.30.0). The v0 tag would then be updated whenever a new tag is created. This would allow us to have the latest version setup without extra maintenance.

Tags on Github are by default mutable, you will have to override it with git push --tags --force, that is correct.

It is not my place to decide on what you choose, however my 2 cents would be that supporting major version tags rather than latest would be preferable. This is the same pattern as how Github does their own actions tagging, e.g. https://github.com/actions/checkout/tags . This is mostly done to avoid disruption with sudden major version bumps.

Alternatively to support a true latest version you could support inputting branch refs rather than only tag versions(?) This way we could refer to the master branch, or for those who want immutable rulesets could refer to git SHAs, e.g.

## References

Issue copied from: https://github.com/terraform-linters/tflint-ruleset-azurerm/issues/462

**Comment by wata727** [comment]
In my opinion, I'm not a fan of unfixed version tags because they cause supply chain attack risk.
While such a mechanism may be useful, I believe that the official features should be as safe as possible by default, and it would be difficult to officially provide such a feature.

See also https://github.com/terraform-linters/tflint/issues/1181 about the `latest` tag discussions.
See also https://www.wiz.io/blog/new-github-action-supply-chain-attack-reviewdog-action-setup

My suggestion is to use an automated tool like Renovate to keep it updated automatically.
